pages/exercises/[lessonSlug].tsx

```diff
@@ -13,20 +13,55 @@ import NavCard from '../../components/NavCard'
 import ExercisePreviewCard, {
   ExercisePreviewCardProps
 } from '../../components/ExercisePreviewCard'
+import { NewButton } from '../../components/theme/Button'
+import ExerciseCard, { ExerciseCardProps } from '../../components/ExerciseCard'
+import { ArrowLeftIcon } from '@primer/octicons-react'
+import { Text } from '../../components/theme/Text'
 
 const exampleProblem = `const a = 5
 a = a + 10
 // what is a?`
 
+const exampleProblem2 = `let a = 5
+a = a + 10
+// what is a?`
+
+const exampleProblem3 = `let a = 5
+a = a + '10'
+// what is a?`
+
 const mockExercisePreviews: ExercisePreviewCardProps[] = [
   { moduleName: 'Variables', state: 'ANSWERED', problem: exampleProblem },
   { moduleName: 'Variables', state: 'NOT ANSWERED', problem: exampleProblem },
   { moduleName: 'Variables', state: 'ANSWERED', problem: exampleProblem }
 ]
 
+const mockExercises: ExerciseCardProps[] = [
+  {
+    challengeName: 'Variable mutation',
+    problem: exampleProblem,
+    answer: 'An error is thrown',
+    explanation: 'You cannot reassign variables that were created with "const".'
+  },
+  {
+    challengeName: 'Variable mutation 2',
+    problem: exampleProblem2,
+    answer: '15',
+    explanation: 'You can reassign variables that were created with "let".'
+  },
+  {
+    challengeName: 'Variable mutation 3',
+    problem: exampleProblem3,
+    answer: `'510'`,
+    explanation:
+      'If you add a string and number together, the number gets converted to a string and then they get concatenated.'
+  }
+]
+
 const Exercises: React.FC<QueryDataProps<GetAppQuery>> = ({ queryData }) => {
   const { lessons, alerts } = queryData
   const router = useRouter()
+  const [exerciseIndex, setExerciseIndex] = useState<number>(-1)
```

Comment:
Will something throw an error if we didn't specify the type? If no, I think it can be removed because TS is capable of inferring the type in this case.

Reply:
Yeah, good catch, I can remove the type parameter so it's just `useState(-1)`.

Reply:
Done.